ormconfig.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
@war1oc CC @asad-mlbd 
Sorry guys if this is a trivial question, but any reason why we aren't keeping this file as a `json` config as shown in the NestJS docs? https://docs.nestjs.com/techniques/database
> Alternatively, rather than passing a configuration object to `forRoot()`, we can create an `ormconfig.json` file in the project root directory.

Reply:
Didn't use json so that we can use `process.env` and read variables from the environment

Reply:
@war1oc Thanks for clarifying. 
That makes sense. 
Also do we need to worry about this point at all?
> Note that the `ormconfig.json` file is loaded by the typeorm library. Thus, any of the extra properties described above (which are supported internally by way of the `forRoot()` method - for example, `autoLoadEntities` and `retryDelay`) won't be applied.

Reply:
Resolved from my side. 
Let's proceed with two separate instances of config definitions.